temporalio/worker/worker.py

```diff
@@ -359,3 +373,82 @@ class WorkerConfig(TypedDict, total=False):
     graceful_shutdown_timeout: timedelta
     shared_state_manager: Optional[SharedStateManager]
     debug_mode: bool
+
+
+_default_build_id: Optional[str] = None
+
+
+def load_default_build_id(*, memoize: bool = True) -> str:
```

Comment:
This was basically copied verbatim from `workflow_service.py`. Heavy scrutiny not needed (but welcome).